LEAF_Request_Portal/form.php

```diff
@@ -136,6 +136,33 @@ public function getFullForm($recordID, $limitCategory = null)
         return $fullForm;
     }
 
+    /**
+     * Copy of getFullForm, but returns sub-indicators as array to preserve order
+     * @param int $recordID
+     * @param string $limitCategory
+     * @return array
+     */
+    public function getFullFormPrint($recordID, $limitCategory = null)
```

Comment:
Instead of creating a new function here, just add a param and conditional to getFullForm() to avoid repeating code.